qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -642,13 +651,15 @@ def on_mode_left(self, mode):
             if widget is None:
                 return
             widget.setFocus()
+        elif config.val.tabs.mode_on_change == 'restore':
```

Comment:
The `elif` means the input mode won't be set correctly when `if mode in [usertypes.KeyMode.command, usertypes.KeyMode.prompt, usertypes.KeyMode.yesno]:` is true. I assume that wasn't intended?